package.json

```diff
@@ -6,9 +6,9 @@
     "build": "npm run lint && tsc",
     "start": "npm run lint -- --force && run-p lint:watch build:watch",
     "test": "npm run test:all",
-    "test:integration": "jest --config jest/integration.config.js",
-    "test:unit": "jest --config jest/unit.config.js",
-    "test:all": "jest",
+    "test:integration": "node env.js && jest --config jest/integration.config.js",
```

Comment:
calling `node env.js` in three different tasks is kinda [DRY](https://en.wikipedia.org/wiki/Don't_repeat_yourself)
let's extract to a standalone task for maintainability